app/controllers/workflows_api_client/application_base_controller.rb

```diff
@@ -8,11 +8,17 @@ def request_headers
       {
         headers: {
           'Utility-Id': utility_id,
-          'Content-Type': request.headers['Content-Type']
+          'Content-Type': request.headers['Content-Type'],
+          'Channel': channel
         }.compact
       }
     end
 
+    def channel
+      return super if defined?(super)
+      nil
+    end
```

Comment:
Sería lo mismo hacer?
```suggestion
    def channel
      super if defined?(super)
    end
```